forem/forem
# Asynchronous organization delete

## What type of PR is this? (check all applicable)


## Description
Made organization delete asynchronous so that the user won't be stuck while deleting. The user will be notified by email after an org is deleted.
The next step will be refactoring and optimizing the deletion process.

## Related Tickets & Documents
#12575 

## QA Instructions, Screenshots, Recordings
Delete organization on `/settings/organization` page. The user should be notified that the org delete is scheduled, the worker should be scheduled. After the worker was executed, the org should be deleted, and the user should be notified that the org was deleted.

## Added tests?

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [x] This change does not need to be communicated, and this is why not: no major changes in app logic.
